libs/tiled-bitmap/src/layeroperations.cc

```diff
@@ -209,6 +209,52 @@ void CommonOperations::draw(cairo_t* cr, const ConstTile::Ptr tile,
   }
 }
 
+PipetteLayerOperations::PipetteColor PipetteCommonOperations::sumPixelValues(Scroom::Utils::Rectangle<int> area, const ConstTile::Ptr tile, int bps, int spp)
+{ 
+  Scroom::Bitmap::SampleIterator<const uint8_t> si(tile->data.get() + area.getTop(), 0, bps);
```

Comment:
I doubt the computation of the starting point is correct. How about adding some tests?

Reply:
My bad. Indeed this is an error. I seem to have forgotten about scaling the vertical offset by `tile->width` and adding the horizontal offset. I think it should be `tile->data.get() + area.getTop() * tile->width + area.getLeft()`. Sorry about that...

Reply:
Actually, there were some problems here with 1bps that i did not realize. These should be fixed now 😃